homeassistant/components/fritzbox/sensor.py

```diff
@@ -153,7 +142,7 @@ def value_scheduled_preset(device: FritzhomeDevice) -> str:
         device_class=SensorDeviceClass.CURRENT,
         state_class=SensorStateClass.MEASUREMENT,
         suitable=lambda device: device.has_powermeter,  # type: ignore[no-any-return]
-        native_value=value_electric_current,
+        native_value=lambda device: round((device.current or 0.0) / 1000, 3),
```

Comment:
Maybe we should not round here, as rounding can now be configured by the user. 🤔

Reply:
I think, we should not change the behaviour in this PR, but in a new one for all occurrences of `round`.

Reply:
Yes, that's better.